lib/rewardable/Limiter.cpp

```diff
@@ -33,6 +35,37 @@ Rewardable::Limiter::Limiter()
 
 Rewardable::Limiter::~Limiter() = default;
 
+bool Rewardable::Limiter::empty() const
```

Comment:
Perhaps instead implement operator == and then implement empty as comparison with empty limiter?